index.js

```diff
@@ -16,15 +12,41 @@ let lastEnemyId = -1;
 
 let rgValidator = new RGValidator();
 
+/**
+ * Outline of our bot algorithm. For every tick:
+ *  - If the bot is standing on the switch, do nothing
+ *  - If the bot is not near the player, move within range of the player
+ *  - If an enemy is within a certain distance of a player, attack that enemy
+ *  - If the switch is within a range of 5 units from the bot, move onto the switch
```

Comment:
Commented distances don't match actual code, keep in mind these are distance^2 values (looks like you already considered that :D )

Reply:
Will fix!

Reply:
Fixed! Yeah this is why I put units, which I am thinking in my head as "squared SOMETHING" but idk what they even call the distance in Unity